pkg/kubelet/prober/worker_test.go

```diff
@@ -232,12 +258,20 @@ func getTestPod(probeSpec api.Probe) api.Pod {
 	return pod
 }
 
-type CrashingProber struct{}
+type crashingExecProber struct{}
 
-func (f CrashingProber) ProbeLiveness(_ *api.Pod, _ api.PodStatus, c api.Container, _ kubecontainer.ContainerID, _ int64) (probe.Result, error) {
-	panic("Intentional ProbeLiveness crash.")
+func (p crashingExecProber) Probe(_ exec.Cmd) (probe.Result, string, error) {
+	panic("Intentional Probe crash.")
 }
 
-func (f CrashingProber) ProbeReadiness(_ *api.Pod, _ api.PodStatus, c api.Container, _ kubecontainer.ContainerID) (probe.Result, error) {
-	panic("Intentional ProbeReadiness crash.")
+// Format probe type.
+func (t probeType) String() string {
```

Comment:
Would this be useful in the non-test code? Maybe we should move this to probe/manager.go.